steps/github-step-repositories-describe/spec.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"type": "object",
+	"properties": {
+		"token": {
```

Comment:
In your example you use `connection: ...` but here you use `token: ...`. To keep a very long story short, we actually do have a GitHub connection that we could _potentially_ use for steps but the required linkage isn't in place yet (because it uses OAuth 2 instead of form-based input). That said, it won't be difficult to get the structure in place that we need now, and people will be able to switch to using the connection in the future. So I would recommend making this:

```json
"connection": {
  "type": "object",
  "description": "A GitHub connection",
  "properties": {
    "accessToken": {
      "type": "string",
      "writeOnly": true,
      "description": "A GitHub access token"
    }
  },
  "required": [
    "accessToken"
  ]
}
```